Database/Corrections/cataQuestFixes.lua

```diff
@@ -1727,6 +1727,9 @@ function CataQuestFixes.Load()
             [questKeys.preQuestGroup] = {14370,14371},
             [questKeys.objectives] = {{{36297,nil,Questie.ICON_TYPE_TALK}}},
         },
+        [14381] = { -- Cleansing Our Crevasse
+            [questKeys.objectives] = {{{90},{35605},{35606}}},
```

Comment:
This is not correct.

The quest has 4 "monster" objectives as per wowhead: https://www.wowhead.com/cata/quest=14381/cleansing-our-crevasse (which can be trusted in 99% of the cases).

Also the order matters.

Reply:
https://github.com/Questie/Questie/pull/6211/commits/1104054bdb0712e05a6d93b448266a6496115d83